library/std/src/sys/vxworks/ext/fs.rs

```diff
@@ -224,7 +224,7 @@ impl FileExt for fs::File {
 
 /// Unix-specific extensions to [`fs::Permissions`].
 ///
-/// [`fs::Permissions`]: ../../../../std/fs/struct.Permissions.html
+/// [`fs::Permissions`]: crate::fs::Permissions
```

Comment:
`fs` is in scope here (`impl FileExt for fs::File`) so the link will be resolved automatically.
```suggestion
```